components/layout.js

```diff
@@ -37,7 +38,7 @@ const Wrapper = glamorous.div((props, {fonts, colors}) => ({
   },
 }))
 
-function Layout({pathname, children, locale}) {
+function Layout({pathname, children, locale, content: {contributors}}) {
```

Comment:
My bad forgot to clean up, this was at first like you're mentioning, changed it to content to also get the dirty `contributorsTitle`.